internal/cmd/local/command_kafka_start.go

```diff
@@ -188,3 +194,20 @@ func getContainerEnvironmentWithPorts(ports *v1.LocalPorts) []string {
 		fmt.Sprintf("KAFKA_REST_LISTENERS=http://0.0.0.0:%s", ports.KafkaRestPort),
 	}
 }
+
+func checkMachineArch() error {
+	cmd := exec.Command("uname", "-m")
+	output, err := cmd.Output()
+	if err != nil {
+		return err
+	}
+	systemArch := strings.TrimSpace(string(output))
+	if systemArch == "x86_64" {
+		systemArch = "amd64"
+	}
+	fmt.Println(systemArch, runtime.GOARCH)
+	if systemArch != runtime.GOARCH {
+		return errors.New(fmt.Sprintf("Running %s Confluent CLI on a %s machine. Please download the correct version.", runtime.GOARCH, systemArch))
```

Comment:
```suggestion
		return errors.New(fmt.Sprintf(`Binary architecture "%s" does not match system architecture "%s"`, runtime.GOARCH, systemArch))
```

Reply:
"Please download the correct version" belongs in a suggestion, not the error text!